src/core/core.datasetController.js

```diff
@@ -967,6 +967,8 @@ class DatasetController {
 
 	/**
 	 * Utility for determining if `options` should be included in the updated properties
+	 * @param {string} mode
+	 * @param {{ target: any; options: any; }} sharedOptions
```

Comment:
I'm surprised by the semi-colons. I did `{{ target: any, options: any }}` in the places I was using this syntax

Reply:
That's how VSCode generates those. Probably configurable somewhere.